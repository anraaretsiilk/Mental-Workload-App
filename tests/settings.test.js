import React from 'react';
import renderer from 'react-test-renderer';
import Settings from '../screens/settings';

test('renders settings correctly', () => {
  const tree = renderer.create(<Settings />).toJSON();
  expect(tree).toMatchSnapshot();
});