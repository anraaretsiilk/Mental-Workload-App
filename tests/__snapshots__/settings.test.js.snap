// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders settings correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 20,
    }
  }
>
  <Text>
    Settings
  </Text>
</View>
`;
